server/planning/events/events_schema.py

```diff
@@ -195,7 +195,30 @@
         "mapping": {"properties": {"qcode": not_analyzed, "name": not_analyzed}},
     },
     # Content metadata
-    "subject": metadata_schema["subject"],
+    "subject": {
```

Comment:
We should also have the same schema for Planning's [subject field](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/planning/planning.py#L1834)

Reply:
alright, I will update it, 
Thanks